### PR TITLE
Show login and password fields at the same time

### PR DESCRIPTION
**Pull request by Marioneq4958**
closes: #66, closes: #43

### CODE REVIEW

**Comment by Marioneq4958** [comment]
Dziennik logowanie:
Zrzut ekranu z 2024-09-13 o 21 04 47

eduVULCAN:
Zrzut ekranu z 2024-09-13 o 21 05 33

**Review by banocean — CHANGES_REQUESTED** [review]
Doesn't hide "Krok 1 z 2" as visible in https://github.com/banocean/ifv/pull/69#issuecomment-2349952827

**Comment by banocean** [comment]
> closes: #66

I think it closes https://github.com/banocean/ifv/issues/43 as well

**Comment by Marioneq4958** [comment]
> Doesn't hide "Krok 1 z 2" as visible in [#69 (comment)](https://github.com/banocean/ifv/pull/69#issuecomment-2349952827)

Fixed

Zrzut ekranu

File: patches/loginPasswordTogether/moveLinksEV.js
Comment (banocean) [inline patches/loginPasswordTogether/moveLinksEV.js:1]: image
There is some more stuff that requires same thing

File: patches.json
Comment (banocean) [inline patches.json:1]: Seems to break when you try to submit invalid password (this also triggers captcha to display)
image

Comment (banocean) [reply patches.json:1]: You can even see animation of the input disappearing :sob:

Comment (banocean) [inline patches.json:1]: Login button redirects me to https://dziennik-logowanie.vulcan.net.pl/warszawazoliborz/Home/Index instead of logging me in

**Review by banocean — CHANGES_REQUESTED** [review]
input should have `autocompleate` attribute to help password managers that work based on official standards instead of trying to find information based on their own

**Review by rotkiW285 — CHANGES_REQUESTED** [review]
My password manager (bitwarden) still does not autocomplete login in dziennik vulcan (chrome password manager works)

**Comment by banocean** [comment]
> My password manager (bitwarden) still does not autocomplete login on pc in dziennik vulcan (chrome password manager works).

Firefox password manager autocomplete works only with password input, login is still not detected for me 😦

**Review by PanLiszka — APPROVED** [review]
U mnie działa

**Comment by rotkiW285** [comment]
Domyślny menadżer haseł firefox wciąż nie widzi inputu hasło na `dziennik-logowanie.vulcan.net.pl`. Bitwarden tak jak nie uzupełniał loginu tak wciąż tego nie robi. Menadżer haseł chrome i cała strona eduvulcan działają perfekcyjnie

**Review by banocean — APPROVED** [review]

**Review by rotkiW285 — APPROVED** [review]
LGTM

**Pull request merged**